hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,5 @@
+RELEASE_TYPE: patch
+
+This patch replaces the deprecated use of
```

Comment:
Our release notes try to focus on what the user needs to know about the release, so there's a bit too much detail here.  I'd just say 

    This patch updates the Hypothesis pytest plugin to avoid a recently 
    deprecated hook interface.  There is no user-visible change.

Reply:
Thank you again; I've replaced my text with your more elegant solution.